client/src/components/Messages.jsx

```diff
@@ -12,10 +12,22 @@ import Message from './Message.jsx';
 import UserContacted from './UserContacted.jsx';
 import dummyData from '../../../messagesDummyData';
 
+import { AuthContext } from '../contexts/index.jsx';
+
+
 function Messages({ userId }) {
   const [usersContacted, setUsersContacted] = useState(dummyData.usersContacted);
   const [messages, setMessages] = useState(dummyData.messages);
   const [userContactedId, setUserContactedId] = useState(null);
+  //  alex test
+  // Context
+  const { getUser } = useContext(AuthContext);
+
+  // getUser('1NsyaeVqmEIcvrcdH9AQ')
```

Comment:
Maybe we can wait until Alex's PR has been merged, because I think he removed these comments